cli/src/logger/mod.rs

```diff
@@ -20,17 +20,18 @@ pub enum ErrorType {
   NoDnsPrivateIp = 119,
   ScaleFailed = 120,
   PostFailed = 121,
+  RunAgzFailed = 122,
 }
 
 #[macro_export]
 macro_rules! error {
   ($errCode:expr, $($message:tt)+) => {async{
-      eprintln!($($message)+);
-      crate::deploy::client_error($errCode, &format!($($message)+)).await;
+    eprintln!($($message)+);
+    deploy::client_error($errCode, &format!($($message)+)).await;
```

Comment:
`crate` was fine, but it was missing the symbol to indicate that it's the `crate` that the macro is in, not the `crate` where the macro was used. So if you instead do:
```suggestion
    $crate::deploy::client_error($errCode, &format!($($message)+)).await;
```
Then you don't need to `use crate::deploy` wherever you use the `error!` macro